common/txmgr/broadcaster.go

```diff
@@ -222,16 +222,15 @@ func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) star
 	eb.wg = sync.WaitGroup{}
 	eb.wg.Add(len(eb.enabledAddresses))
 	eb.triggers = make(map[ADDR]chan struct{})
+	eb.sequenceLock.Lock()
+	eb.nextSequenceMap = eb.loadNextSequenceMap(eb.enabledAddresses)
```

Comment:
I feel like this should probably take a context from the initial `Start` call, no?

Reply:
Updated.